### PR TITLE
Update tutorial-projects-solutions.md

### PR DESCRIPTION
**Pull request by bergano65**

### CODE REVIEW

**Comment by PRMerger8** [comment]
@bergano65 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/get-started/tutorial-projects-solutions.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ We'll start our exploration by creating an empty solution. After you get to know
 
 ::: moniker range="vs-2017"
 
-1. Open Visual Studio.
+1. Open Visual Stud.io.
```
Comment (bergano65) [inline docs/get-started/tutorial-projects-solutions.md]: pls, pay no attention, my typo

File: docs/get-started/tutorial-projects-solutions.md [hunk 2]
```diff
@@ -252,7 +252,7 @@ If you want to check that your unit test is working, choose **Test** > **Run** >
 ![Test Explorer in Visual Studio showing passed test](media/tutorial-projects-test-explorer.png)
 
 > [!TIP]
-> If **Test Explorer** doesn't open automatically, open it by choosing **Test** > **Windows** > **Test Explorer** from the menu bar.
+> If **Test Explorer** doesn't open automatically, open it by choosing **Test** > **Test Explorer** from the menu bar.
```
Comment (bergano65) [inline docs/get-started/tutorial-projects-solutions.md]: there  is no "window"under "test"

**Comment by PRMerger13** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/get-started/tutorial-projects-solutions.md (hunk 1 above)
Comment (ktoliver) [reply docs/get-started/tutorial-projects-solutions.md]: fixed

**Comment by TerryGLee** [comment]
#sign-off

**Pull request merged**